docker/scripts/ci-docker.sh

```diff
@@ -66,18 +66,14 @@ build_exec_image() {
   setup_scratch_dir $SCRATCH_DIR
 
   # need to copy artifacts locally
-  TOOLS_FILE="$OUTPUT_DIRECTORY/heron-tools-install.sh"
-  TOOLS_OUT_FILE="$SCRATCH_DIR/artifacts/heron-tools-install.sh"
+  TOOLS_FILE="$OUTPUT_DIRECTORY/heron-install.sh"
```

Comment:
variable name`INSTALL_FILE` makes more sense. And also same the followings

Reply:
cool. thx.